Documentation/k8s-block.md

```diff
@@ -14,10 +14,33 @@ This guide assumes you have created a Rook cluster as explained in the main [Kub
 
 ## Provision Storage
 
-Before Rook can start provisioning storage, a StorageClass and its storage pool need to be created. This is needed for Kubernetes to interoperate with Rook for provisioning persistent volumes. The [rook-storageclass.yaml](https://github.com/rook/rook/blob/master/demo/kubernetes/rook-storageclass.yaml) sample will create the storage pool automatically. For more options on pools, see the documentation on [creating storage pools](pool-tpr.md).
+Before Rook can start provisioning storage, a StorageClass and its storage pool need to be created. This is needed for Kubernetes to interoperate with Rook for provisioning persistent volumes. For more options on pools, see the documentation on [creating storage pools](pool-crd.md).
+
```

Comment:
Yeah, i'll make sure there is consistency for creating the yaml. I was assuming the `create -f` was implied, but that's not a safe assumption and I see documentation in other repos including that step as well.